enderio-machines/src/main/java/com/enderio/machines/client/gui/screen/AlloySmelterScreen.java

```diff
@@ -31,14 +31,14 @@ public AlloySmelterScreen(AlloySmelterMenu pMenu, Inventory pPlayerInventory, Co
     @Override
     protected void init() {
         super.init();
-        addRenderableOnly(new ProgressWidget(this, () -> menu.getBlockEntity().getProgress(), getGuiLeft() + 56, getGuiTop() + 36, 14, 14, 176, 0));
-        addRenderableOnly(new ProgressWidget(this, () -> menu.getBlockEntity().getProgress(), getGuiLeft() + 104, getGuiTop() + 36, 14, 14, 176, 0));
+        addRenderableOnly(new ProgressWidget(this, () -> menu.getBlockEntity().getProgress(), getGuiLeft() + 56, getGuiTop() + 36, 14, 14, 176, 0, ProgressWidget.Direction.BOTTOM_UP));
```

Comment:
I'd like to explore a cleaner screen/menu system. Might be worth looking into in a future PR.